README.md

```diff
@@ -1,7 +1,21 @@
-# Rust Events team
+# 🗓 Rust Events team
 
-This is the repository for all planning of the Rust event team. We give hints, help and contacts for running events.
+This is the repository for any planning of the [Rust events team](https://github.com/rust-community/team#%EF%B8%8F-events). If you are running an event we would love to offer some tips and contacts along with our experience to make it as smooth as possible.
 
-Feel free to have a look at our [open issues](https://github.com/rust-community/events/issues).
+## Material
 
-If you want to contact us, please send an email to [community@rust-lang.org](mailto:community@rust-lang.org).
+- **Meetups:** Open an issue using [this template](https://github.com/rust-community/events-team/blob/master/ISSUE_TEMPLATE/new_meetup.md) ❤️
```

Comment:
We can use this link here to directly open a new issue using the template:
https://github.com/rust-community/events-team/issues/new?template=new_meetup.md